### ISSUE TITLE
Workshop Publishing Problems & Broken TModLoader UI and Publish Page

### ISSUE THREAD

**Issue by DiamondWysteria** [bluemagic123/tModLoader#4027]
### Version

Latest 1.4.4-stable

### OS

Windows

### Platform

Steam

### This bug affects

Mod capability as a Modder

### Description

I was porting my mod to 1.4.4 and when I tried publishing my mod, the UI was broken. It was offset to be out of reach with 1920x1080 resolution. However, this isn't even the main problem. I was able to access the publish button by changing resolution, but the main error lies in the publishing. When I published my mod, my icon wasn't detected, nor were any of my previous publishing options. I thought maybe I just needed to plug in my icon.png and select the options again, but when I published, it created a separate workshop item. I have the same internal name for my mod. Could this have to do something with a steam workshop agreement error that I agreed to while publishing? Im extremely worried because I think my mod might be unlinked from the main steam workshop page, due to the broken UI and broken publish page.

### Log File

[22:03:58.104] [Main Thread/INFO] [tML]: No Mod on Workshop with internal name: WysteriaMod

(This doesn't make sense though because I've always had the internal name WysteriaMod???)
![image](https://github.com/tModLoader/tModLoader/assets/141670344/cc4620e9-b44f-489e-9e50-b4c8cb27f578)

### Steps to reproduce

1. Build & Reload Mod
2. Publish

PUBLISH PAGE WITH NORMAL RESOLUTION:
![image](https://github.com/tModLoader/tModLoader/assets/141670344/4ee8327e-e4a7-4995-80dd-5e8cec7b5a06)

It showing no options chosen and mod icon empty:
![image](https://github.com/tModLoader/tModLoader/assets/141670344/47aeb1c5-3bcf-415d-a2dd-bcd094b69f2b)

### Expected Behavior

Because the mod has the same internal name, it should work completely fine, everything should be detected, but it seems like tModLoader is extremely buggy as the UI is offset and nothing was detected.

### Actual Behavior

Every time I tried to publish the mod, everything reset, and once I published again, it created a new workshop entry every time.

### Additional Information

Steam workshop license agreement issues? I'm not sure if that had anything to do with it though since i agreed to the new one while publishing the first time.
Basically, the crux of the issue is, tModLoader isn't recognizing my mod for some reason, and is making new workshop entries every time I try to publish the same source mod, which has always had WysteriaMod as its internal name??? It's not updating the existing version or workshop item.

**Comment by JavidPack** [comment]
Please post the actual log files. I suggest deleting the logs folder, then launching tModLoader and attempting to go to the publish page. Once you get to the off-centered publish page, post a comment here with all the logs file attached directly.

**Comment by Cyrillya** [comment]
I am having the same issue
Log file:
[client.log](https://github.com/tModLoader/tModLoader/files/14072507/client.log)

**Comment by JavidPack** [comment]
This is partially addressed in 7dada24f2fe5c7fda8e32a9d06018b71985ba695, we'll need to revisit this next stable to see what else needs to be fixed.

**Comment by JavidPack** [comment]
I believe this is fixed, feel free to reopen if still experiencing this.
